commit f41d5f969b94d3bafea56f9fcfc7db5e5d1e3f5f
Author: Ralf Jung <post@ralfj.de>
Date:   Sun Mar 10 11:12:11 2013 +0100

    Fix parse error in advanced memroy information for 3.8 kernels
    
    REVIEW: 109385
    BUG: 314818
    FIXED-IN: 4.10.2

diff --git a/libs/ksysguard/scripts/smaps/main.js b/libs/ksysguard/scripts/smaps/main.js
index fb938c4..271864f 100644
--- a/libs/ksysguard/scripts/smaps/main.js
+++ b/libs/ksysguard/scripts/smaps/main.js
@@ -70,9 +70,8 @@ function parseSmaps() {
             dataBlock.dev = headingMatch[4];     /* Device (major,minor) */
             dataBlock.inode = headingMatch[5];   /* inode on the device - 0 means no inode for the memory region - e.g bss */
             dataBlock.pathname = headingMatch[6];
-        } else if(smaps[i] !== "") {
-            throw("Could not parse '" + smaps[i]) + "'";
         }
+        // ignore unknown lines
     }
     if(dataBlock)
         data.push(dataBlock);
